packages/pyright-internal/src/analyzer/typeEvaluator.ts

```diff
@@ -19953,6 +19953,19 @@ export function createTypeEvaluator(importLookup: ImportLookup, evaluatorOptions
                                 .getDeclarations()
                                 .find((decl) => decl.type === DeclarationType.Parameter);
                         }
+
+                        const parameterDetails = getParameterListDetails(type);
```

Comment:
It felt slightly heavy-handed to call the entire `getParameterListDetails` here but also the code to deduce is `unpackedKwargsTypedDictType` is not the most trivial.

Reply:
Looks reasonable to me.